docs/collections.md

```diff
@@ -41,13 +122,15 @@ This will place this `mypost.md` into the `post` collection with all other piece
 ```js
 module.exports = function({collections}) {
   return `<ul>
-${collections.post.map((post) => `<li>${ post.data.title }</li>`).join("\n")}
-</ul>`;
+    ${collections.post.map((post) => `<li>${ post.data.title }</li>`).join("\n")}
```

Comment:
This should be `collections.posts` right?

Reply:
Correct.